mindmeld/models/taggers/pytorch_crf.py

```diff
@@ -287,7 +287,7 @@ def save_best_weights_path(self, path):
         Args:
             path (str): Path to save the best model weights.
         """
-        self.best_model_save_path = path
+        self.best_model_save_path = os.path.abspath(path)
```

Comment:
Does this block models from being archived/copied?

Reply:
Fixed that by changing the way pyTorch-CRF loads and dumps models